### ISSUE TITLE
.NET: Operators `>`, `<` break SK's plan

### ISSUE THREAD

**Issue by Escoto** [microsoft/semantic-kernel#2047]
**Describe the bug**
Trying to create a plan that can take user's conditions into consideration, SK's fails to parse the plan from xml.
`PlanningException: Invalid plan: Plan parsing error, invalid XML`.

**To Reproduce**
1. I took this as starting point [semantic kernel docs sample](https://github.com/MicrosoftDocs/semantic-kernel-docs/blob/main/samples/dotnet/04-Planner/Program.cs)
2. Then I added a new `Plugin` / `SKFunction` capable of "sending emails".
3. Using the asks: 
```
var ask1 = "Only when the result of 10 times 10 is less than 50 send an email saying: Hello!!. Do not send an email if the answer is more than 50.";
var ask2 = "Only when the result of 10 times 10 is equal to 50 send an email saying: Hello!!. Send an email only if the answer is exactly 50.";
```
4. I got these errors respectively
```
Unhandled exception. Microsoft.SemanticKernel.Planning.PlanningException: Invalid plan: Plan parsing error, invalid XML
 ---> Microsoft.SemanticKernel.Planning.PlanningException: Invalid plan: Failed to parse plan xml.
 ---> System.Xml.XmlException: '<', hexadecimal value 0x3C, is an invalid attribute character. Line 3, position 64.
```

```
Unhandled exception. Microsoft.SemanticKernel.Planning.PlanningException: Invalid plan: Plan parsing error, invalid XML
 ---> Microsoft.SemanticKernel.Planning.PlanningException: Invalid plan: Failed to parse plan xml.
 ---> System.Xml.XmlException: '<' is an unexpected token. The expected token is '='. Line 36, position 63.
```

**Expected behavior**
SK plan can handle operators (`<`, `>`, etc...) without breaking.

**Platform**
 - OS: Windows
 - IDE: VS Code
 - Language: C#
 - Source: SemanticKernel Version="0.15.230531.5-preview"

**Comment by lemillermicrosoft** [comment]
There were fixes in 0.16 and 0.17 specific to this kind of issue. Please try with those or latest.

Question:
1) What model are you using?
2) What is the LLM response (it may just not be valid response -- 0.16+ make it easier to debug these details).

**Comment by lemillermicrosoft** [comment]
Closing for now with no repro available. Please file new issue or re-open as appropriate. Thanks!
